uniffi_bindgen/src/backend/types.rs

```diff
@@ -58,6 +58,18 @@ pub trait CodeType {
         unimplemented!("Unimplemented for {}", self.type_label(oracle))
     }
 
+    /// Name of the FfiConverter
+    ///
+    /// This is the object that contains the lower, write, lift, and read methods for this type.
+    /// Depending on the binding this will either be a singleton or a class with static methods.
+    ///
+    /// This is the newer way of handling these methods and replaces the lower, write, lift, and
+    /// read CodeType methods.  Currently only used by Kotlin, but the plan is to move other
+    /// backends to using this.
+    fn ffi_converter_name(&self, oracle: &dyn CodeOracle) -> String {
+        oracle.class_name(&format!("FfiConverter{}", self.canonical_name(oracle)))
+    }
```

Comment:
Do any types _not_ cross the FFI? (objects didn't used to).

Reply:
I think `Error` only goes one way, it can't be passed back to rust.  Although that might be fixable that now.  In general, this moves the logic into the the FFIConverter classes.  So if a type can't cross the FFI for some reason, then I think we should have the `FffiConverter` with unimplemented methods.